test/unit/test-layout.cxx

```diff
@@ -85,7 +82,7 @@ TEST(LayoutTest, 2D_JI)
    * (-1, -1), (0, -1), (1, -1)
    * (-1, -2), (0, -2), (1, -2)
    */
-  const my_layout layout({-1,-2}, {1,0});
+  const my_layout layout = my_layout::from_layout({-1,-2}, RAJA::make_permuted_layout<RAJA::PERM_JI>(3,3));
```

Comment:
It would be nice if there was a RAJA::make_offset_layout function that would take the offsets, permutation, and strides.  That would be nice syntactic sugar to avoid this kind of code, which I think is a little opaque.